css/css-backgrounds/border-image-width-009.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Backgrounds and Borders Test: border image area - border-image-width with 1 value with percentage unit</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="http://www.w3.org/TR/css3-background/#the-border-image-width">
+  <link rel="match" href="../reference/ref-filled-green-100px-square.xht">
+
+  <meta name="flags" content="">
+  <meta content="This test checks that the border image area can still exists even if the border area (or border belt) of an element is inexistent. The difference with this test and the border-image-width-005 test is the usage of percentage unit. In this test, the border image fills the content area, the padding area and 25px of the extended-beyond-border-belt area, 25px into the margin area of the element." name="assert">
+
+  <!--
+  created 2015-11-18
+  [css-backgrounds] Please remove the "unless there is a border image" wording
+  http://lists.w3.org/Archives/Public/www-style/2015Nov/0258.html
```

Comment:
You are referencing this bug, but the test is not testing this scenario because you explicitly set the border-width to zero.

Reply:
I have removed the reference to such www-style/2015Nov/0258.html message but I have made the new meta-test 
http://www.gtalbot.org/BrowserBugsSection/CSS3Backgrounds/border-image-width-009-new.html
to specifically check such scenario (a missing border-style inside a border shorthand declaration).